### DISCUSSION TITLE
Copy Model or fields to other Vemto project

### DISCUSSION THREAD

**Discussion by fredyns** [TiagoSilvaPereira/vemto-issues#419] [Ideas & Suggestions]
my Vemto project is broken, but i don't know where.
livewire edit form can't be shown. but works fine for new project.

it would be better if i can copy any Model to new project.
it's a lot.

**Comment by TiagoSilvaPereira** [comment]
Hi @fredyns thank you for your suggestion, it is coming to Vemto 2

**Reply by fredyns** [reply]
thank you for your response. (y)

now i know my problem. 
i should laughing myself for my lack in laravel =)))

livewire edit form won't work for UUID primary key.
```id``` parameter in function ```editModel('{{ $model->id }}')``` must be wrapped in ticks.
please add them in built in template.

**Reply by TiagoSilvaPereira** [reply]
Excellent. Could you please open an issue here so we can track the problem?

https://github.com/TiagoSilvaPereira/vemto-issues/issues
